TRENZ.Lib.RazorMail.Core/Models/MailHeaderCollection.cs

```diff
@@ -170,7 +170,7 @@ private IEnumerable<MailAddress> GetAddresses(string key)
     /// </summary>
     /// <param name="other">The collection to append.</param>
     /// <returns>The current instance.</returns>
-    public MailHeaderCollection Append(MailHeaderCollection other)
+    public MailHeaderCollection AppendFrom(MailHeaderCollection other)
```

Comment:
Hm, `From` seems like poor naming, given that it also has a more specific meaning (see, for example, `DefaultFrom`).